scripts/common

```diff
@@ -4,23 +4,38 @@ trap "exit 1" TERM
 export TOP_PID=$$
 set -e
 
+WORKSPACE="${WORKSPACE-`pwd`}"
+
+# the default (home dir) is fine on Travis, since each jobs gets its own worker (ivy cache is cached by travis)
+IVY_HOME="${IVY_HOME-$HOME/.ivy2}"
+
 # Known problems : does not fare well with interrupted, partial
 # compilations. We should perhaps have a multi-dependency version
 # of do_i_have below
 
 LOGGINGDIR="$WORKSPACE/logs"
 mkdir -p $LOGGINGDIR
 
-unset SBT_HOME
-SBT_HOME="$WORKSPACE/.sbt"
-mkdir -p $SBT_HOME
-IVY_CACHE="$WORKSPACE/.ivy2"
-mkdir -p $IVY_CACHE
-rm -rf $IVY_CACHE/cache/org.scala-lang
+# unset SBT_HOME
+# SBT_HOME="$WORKSPACE/.sbt"
+# mkdir -p $SBT_HOME
+# IVY_CACHE="$WORKSPACE/.ivy2"
+# mkdir -p $IVY_CACHE
```

Comment:
I don't get the full picture why we don't need those anymore; I see the `bootstrap` script (for jenkins) uses `sbtArgs` to set `-ivy/-Dsbt.global.base/-sbt-dir`, but it did that before. So the variables were unused before? Can we just remove them?

Reply:
Yeah, i think they were completely unused for a while. Maybe ide integration used them?

Reply:
I'm going to leave a trace in the comments for now, because it's so hard to convince yourself you didn't overlook a use site.

Reply:
I just realized I could use https://github.com/search?q=org%3Ascala+SBT_HOME&type=Code

nuking the comments